database/seed.js

```diff
@@ -1,108 +1,58 @@
 /* eslint-disable arrow-body-style */
-const mysql = require('mysql');
-const Promise = require('bluebird');
-const { writeRecordsToFile } = require('./seedMethods');
+const { createBook } = require('./sampleDataModel');
+const fs = require('fs');
+const fastCSV = require('fast-csv');
 
-const ONE_MILLION = 1000000;
-
-writeRecordsToFile(ONE_MILLION * 10);
 
+const ONE_MILLION = 1000000;
 
-// const connection = mysql.createConnection({
-//   user: 'root',
-//   password: 'dobby',
-//   host: 'localhost'
-// });
 
-// const db = Promise.promisifyAll(connection, { multiArgs: true });
+const writeRecordsToFile = (size) => {
+  const LABEL = 'CSV data writing';
+  console.time(LABEL);
+  //use '|' delimiter to avoid conflict with fields with commas
+  const csvWriteStream = fastCSV.createWriteStream({delimiter: '|'});
+  const fsWriteStream = fs.createWriteStream("bookDetails.csv");
+  fsWriteStream.on("finish", () => {
```

Comment:
would recommend turning on your linter so you can be consistent w/ small details like `'` vs `"`